exporter/elasticsearchexporter/README.md

```diff
@@ -184,6 +184,16 @@ Settings related to node discovery are:
 
 Node discovery can be disabled by setting `discover.interval` to 0.
 
+### Telemetry settings
+
+The Elasticsearch Exporter's own telemetry settings for testing and debugging purposes.
```

Comment:
Can we mention here that this may expose sensitive data, so users need to be cautious.